### DISCUSSION TITLE
Welcome to subtitler Discussions!

### DISCUSSION THREAD

**Discussion by anupamkumar** [anupamkumar/subtitler#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AnkitSingh-In** [comment]
hi Anupam.
Great tool you've created here. I have installed all necessary files, and the Subtitler-GUI is running on my Win11 PC.

I just have two queries,

1. Is the tool works on audio file too (e.g., Recording.aac)
2. Is 'Large-V3' is only parameter model for tool to work? I was hoping that tool let me choose language model (e.g., tiny, base, small etc.)

If I missed these in Wiki, please forgive me.

**Comment by huang-jenny** [comment]
Just wanted to say thank you for this tool! Works so much better than Premiere's built-in transcription
